misc-scripts/xref_mapping/XrefParser/RefSeqCoordinateParser.pm

```diff
@@ -397,14 +415,14 @@ sub run_script {
 
 # Add link between Ensembl gene and EntrezGene
           if (defined $entrez_ids{$entrez_id} ) {
-            foreach my $dependent_xref_id (@{$entrez_ids{$entrez_id}}) {
+            while (my $dependent_xref_id = shift @{$entrez_ids{$entrez_id}}) {
               $add_dependent_xref_sth->execute($xref_id, $dependent_xref_id);
             }
           }
 
 # Also store refseq protein as direct xref for ensembl translation, if translation exists
           if (defined $tl && defined $tl_of) {
-            if ($tl_of->seq eq $tl->seq) {
+            if (md5($tl_of->seq) eq md5($tl->seq)) {
```

Comment:
what advantage does using the md5 bring?
we still call the seq method, so I am assuming the same query is still being run underneath